src/envelopes.js

```diff
@@ -26,6 +26,14 @@ const errors = {
   FUNDS_RETURNED_FAILED: {
     code: 'envelope-7',
     message: "Error, funds couldn't be returned"
+  },
+  TOTAL_FUNDS_RETURNED_FAILED: {
+    code: 'envelope-8',
+    message: "The total amount of the envelope could not be released"
```

Comment:
Tengo dudas sobre el titulo de esta y el mensaje como que no me hacen match

Reply:
es que hay un punto donde el cliente puede retirar el monto total del sobre pero en ocaciones falla, entonces era como 
Total de fondos devueltos fallados pero no se si ve mejor ERROR_WHEN_RETURNING_TOTAL_FUNDS en el titulo y en la descripción Could not return all available funds